apps/onlinereports/src/components/revive-ad.tsx

```
@@ -0,0 +1,24 @@
+}
+
+export const ReviveAd: React.FC<ReviveAdProps> = ({zoneId, reviveId}) => {
+  useEffect(() => {
+    if ((window as any).reviveAsync) {
+      ;(window as any).reviveAsync[reviveId].refresh()
+    }
+  }, [reviveId])
+
+  return (
+    <div
+      dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)